IntelliTect.Multitool/IPaths.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace IntelliTect.Multitool;
+/// <summary>
+/// Provides normalized paths.
+/// </summary>
+public interface IPaths
```

Comment:
I would propose just making this a static class with a static method rather than keeping it as a default interface member.